### ISSUE TITLE
About fine-tuning resnet-18 on hmdb

### ISSUE THREAD

**Issue by Collinxz** [kenshohara/3D-ResNets-PyTorch#114]
Hi.
I want to fine-tune resnet-18 on hmdb1 with lr=0.001 and weight_decay=0.00001,but after 300 epoch, the val only get 51% accuracy which below 56.4 in your paper.What do i miss? Thank you!

**Comment by huangwei1995** [comment]
I have the same question when fine-tuning.

**Comment by fengwuxuan** [comment]
Me too, and I try to train from scratch on hmdb and ucf101, but the val accuracy is lower than paper's...

**Comment by shuangshuangguo** [comment]
@Collinxz @huangwei1995 @fengwuxuan You can try my scripts to finetune, which I got 53.45% clip-leval accuracy.
https://github.com/shuangshuangguo/3D-ResNet-Pytorch
